.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.6']
+        ruby-version: ['3.2']
```

Comment:
Do we really need a matrix here if we are only going to use 1 version of ruby?

Reply:
Fixed it